src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
```

Comment:
Write informative messages in methods. The message should indicate what type of `Machine` is working right now `Truck`, `Bulldozer` or `Excavator`. Consider specifying that it is a Truck that has started its work, to make the message more informative.